currency.go

```diff
@@ -20,64 +20,81 @@ var currencies = map[string]*Currency{
 	"AFN": {Decimal: ".", Thousand: ",", Code: "AFN", Fraction: 2, Grapheme: "\u060b", Template: "1 $"},
```

Comment:
@danielwitz I see you updated currency list. Is it official list? What is the source?

Reply:
@Rhymond I took it from the ruby's [money gem](https://github.com/RubyMoney/money/blob/master/config/currency_iso.json). It's well tested and commonly used.